src/Components/utils/useUserActions.js

```diff
@@ -30,13 +35,15 @@ export const useUserActions = () => {
             type: "SET_WISHLIST",
             payload: response.data.response,
```

Comment:
- You can use variable instead of hard coded string `SET_WISHLIST` for action types